Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -61,6 +61,7 @@ namespace Platform
 namespace RedirectOutput
 {
     using RedirectOutputFunc = AZStd::function<void(const char*)>;
+    AZStd::string lastPythonError;
```

Comment:
Recommend making this a QStringList or AZ::Vector<AZStd::string> and store all the errors generated during a call by appending them.